modules/svg/rapid_features.js

```diff
@@ -240,12 +241,26 @@ export function svgRapidFeatures(projection, context, dispatch) {
     }
   }
 
+  function transformViewFieldPoint(d) {
+      if(!d.loc) {
+        d.loc = [d.lon, d.lat];
+      }
+      var t = svgPointTransform(projection)(d);
+      t += ' scale(1.5,1.5) rotate(' + Math.floor(d.ca) + ',0,0)';
+      return t;
+  }
+
+  function viewfieldPath() {
+      return 'M 6,9 C 8,8.4 8,8.4 10,9 L 16,-2 C 12,-5 4,-5 0,-2 z';
```

Comment:
cc @bhousel for a general comment: for new svg const like this, would you recommend adding new folder/files under [RapiD/svg](https://github.com/facebookincubator/RapiD/tree/streetview_workflow/svg)?

Reply:
good point! I have copied this from streetside.js in modules/services

Reply:
👍  ah, got it. then it might make more sense to have a separate PR if we decide to put these into new svg files.

Reply:
It's ok to embed tiny svg path strings like this in the code.  
Usually I think it would be good to have standard symbols included in the stylesheet, but viewfields are a bit special because we might want to render them with different width depending on the field of view of the image, so rendering them on the fly with code makes sense too.